evm_loader/lib/src/types/tracer_ch_common.rs

```diff
@@ -73,6 +75,16 @@ pub struct AccountRow {
     pub txn_signature: Vec<Option<u8>>,
 }
 
+#[derive(Row, serde::Deserialize, Clone)]
+pub struct AccountDataRow {
```

Comment:
Why do you need additional type for accounts? It differs only with one `pubkey` field from `AccountRow`
Please, use AccountRow for your purposes (you can add `pubkey` field there)